docs/guide/topics/threads.rst

```diff
@@ -0,0 +1,256 @@
+
+You can view the API reference for creating threads here: :meth:`TextChannel.create_thread` / :meth:`Message.create_thread`.
+
+For a quick primer, let's try making a command that creates a thread and then sends a message in it:
+
+.. code-block:: python3
+
+    @bot.command()
+    async def create_thread(ctx: commands.Context):
+        thread = await ctx.channel.create_thread(name="My cool thread", message=ctx.message)
+
+        await thread.send("Created!")
+
+which will create a thread like so:
+
+.. image:: /images/guide/threads/create_public_thread.png
+
+Simple, right? Let's discuss some of the finer details.
+
+Public threads
+~~~~~~~~~~~~~~~
+
+Public threads can be viewed by anyone who can see the channel they belong to.
+People can browse and join these threads as they please.
+
+Public threads can be tied to a specific message. You can use a message that already exists, but you can also send a new message to attach the thread to.
+
+Let's expand on our example from before by making it possible to name the thread. We can also switch to using the convenient :meth:`Message.create_thread` shortcut.
+
+.. code-block:: python3
+    :emphasize-lines: 3
+
+    @bot.command()
+    async def create_thread(ctx: commands.Context, *, thread_name: str):
+        thread = await ctx.message.create_thread(name=thread_name)
+
+        await thread.send(f"Hello from the {thread_name} thread!")
+
+and then we have our public thread:-
+
+.. image:: /images/guide/threads/create_public_thread_param.png
+
+As discussed before, we can see it from the thread list, even if we're not a member:-
+
+.. image:: /images/guide/threads/not_a_member.png
+    :scale: 90
+
+Public threads can also be standalone, with no message tied to the creation. The example code above would be similar to do this:-
+
+.. code-block:: python3
+    :emphasize-lines: 3
+
+    @bot.command()
+    async def create_thread(ctx: commands.Context, *, thread_name: str):
+        thread = await ctx.channel.create_thread(name=thread_name, type=discord.ChannelType.public_thread)
+
+        await thread.send(f"Hello from the {thread_name} thread!")
+
+You can see here we now utilise ``ctx.channel`` over ``ctx.message`` to create the thread, but we also pass :attr:`ChannelType.public_thread` to the ``type=`` keyword argument to denote that it will be a public thread.
+
+
+Private threads
+~~~~~~~~~~~~~~~~
+
+Private threads exist as "invite only" threads.
+By default, they only contain the person who created them, but other people can be invited to participate.
+
+Private threads can optionally be 'invitable'.
+If a thread is invitable, it means anyone in that thread can invite others to it.
+Otherwise, only the owner and people with the :attr:`~Permissions.manage_threads` permission can add new people to the thread.
+
+Users with :attr:`~Permissions.manage_threads` ("Moderators") can also see a list of private threads they're not in, similar to public threads.
```

Comment:
Yes? The term for someone who can manage threads is a moderator, it doesn't matter what "staff" may be present.

Reply:
Just to avoid confusion, it would be best to just remove it just to explicitly mention the permission that is needed instead. I could see this getting confusing for some